Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -89,18 +89,20 @@ public class BTShopperInsightsClient {
         }
     }
 
-    /// Call this method when the PayPal button has been successfully displayed to the buyer.
+    /// Call this method when the PayPal or Venmo button has been successfully displayed to the buyer.
     /// This method sends analytics to help improve the Shopper Insights feature experience.
     /// - Parameters:
-    ///    - paymentMethodsDisplayed: Optional:  The list of available payment methods, rendered in the same order in which they are displayed i.e. ['Apple Pay', 'PayPal']
-    ///    - experiment: Optional:  A `JSONObject` passed in as a string containing details of the merchant experiment.
-    public func sendPayPalPresentedEvent(paymentMethodsDisplayed: [String?] = [], experiment: String? = nil) {
-        let paymentMethodsDisplayedString = paymentMethodsDisplayed.compactMap { $0 }.joined(separator: ", ")
+    ///     - buttonType: Type of button presented - PayPal, Venmo, or Other
+    ///     - presentmentDetails:  Detailed information, including button order, experiment type, and
+    ///     page type about the payment button that is sent to analytics to help improve the Shopper Insights
+    ///     feature experience.
+    public func sendPresentedEvent(for buttonType: BTButtonType, presentmentDetails: BTPresentmentDetails) {
         apiClient.sendAnalyticsEvent(
-            BTShopperInsightsAnalytics.payPalPresented,
-            merchantExperiment: experiment,
-            paymentMethodsDisplayed: paymentMethodsDisplayedString,
-            shopperSessionID: shopperSessionID
+            BTShopperInsightsAnalytics.buttonPresented,
+            buttonOrder: presentmentDetails.buttonOrder.rawValue,
+            buttonType: buttonType.rawValue,
+            experimentType: presentmentDetails.experimentType.formattedExperiment,
+            pageType: presentmentDetails.pageType.rawValue
```

Comment:
Lets also make sure to include `shopperSessionID: shopperSessionID` in these events. We should be able to add a test around this as well based on: https://github.com/braintree/braintree_ios/pull/1485

Reply:
Not sure how `shopperSessionID` was left out here, thanks!